app/helpers/application_drafts_helper.rb

```diff
@@ -18,6 +18,12 @@ module ApplicationDraftsHelper
                     :application_money,
                     :application_minimum_money]
 
+  def in_same_team_as_other_student?(student)
```

Comment:
Hm, just wondering, can't you reuse `private_application_data` from the `ApplicationDraftsHelper` for that 🤔 

If I understand your motivations here, it should basically be the same...

Reply:
Oh, hm, looks like yes most probs. Thanks for the tip 😅